fli-iam/shanoir-ng
# #2275-processed-dataset-bug

- Correctly download EEG processed dataset
- Correctly display input dataset no processing detail
- Correctly set processed dataset folder when downloading

How to test:
Deploy on OFSEP qualif (please)
- Import a processed dataset (select a nifti) and select EEG as dataset type.

--> Dataset can be downloaded
--> Path in the donwload folder is not undefined-undefined anymore

Go on processing detail:
--> Input datasets are correctly loaded.

Go on examination 52384
--> Check that the loading time of the tree is not >9000 !
